### ISSUE TITLE
Unable to run PHP Apps (on Mac with macOS Monterey)

### ISSUE THREAD

**Issue by NoCommandLine** [NoCommandLine/NoCommandLine-App#2]
When you try to run a PHP App, you get the following in the log

```
Run PHP Project
PID: undefined
Error: spawn php ENOENT
Process with PID undefined was closed with code -2
PHP run program process with PID undefined was closed with code -2
```

**Steps to reproduce**
1. Start NoCommandLine. 
2. Create New Application and select PHP as runtime. 
3. Click the button to run the App.

**Comment by NoCommandLine** [comment]
Generally, ENOENT means the file (executable) can't be found. In this case, it would mean the PHP executable wasn't located in the `$PATH` variable 

It looks like PHP is [no longer installed](https://developer.apple.com/forums/thread/681907) by default in Mac OS 12+ (Monterey, Ventura, etc). This means it isn't available in `$PATH` variable i.e. running

`which php `

**doesn't** produce

`/usr/bin/php  ` 

### Immediate Fix
Add the path to your PHP installation to `$PATH `variable


### Long-term Fix
We will provide an attribute (input field) where you can provide the location of your PHP installation (similar to what we do for Python and Go; see this [example](https://nocommandline.com/help/#error_missing_command)). 

When you try to run a PHP App, we will check if you have provided a custom location for your PHP script. If you have, we will use it else we will default to the locations in `$PATH` variable.

**Comment by NoCommandLine** [comment]
Fixed in V2.0.8

On the Preferences page, there is now an option to provide the path to your PHP executable file.

To get to Preferences Page

1. **Mac:** Menu > NoCommandLine > Preferences
2. **Windows:** Menu > Preferences
